lexicons/app/bsky/label/defs.json

```diff
@@ -0,0 +1,103 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.label.defs",
+  "defs": {
+    "labelerView": {
+      "type": "object",
+      "required": ["uri", "cid", "did", "creator", "displayName", "indexedAt"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "did": { "type": "string", "format": "did" },
+        "creator": { "type": "ref", "ref": "app.bsky.actor.defs#profileView" },
+        "displayName": { "type": "string" },
+        "description": {
+          "type": "string",
+          "maxGraphemes": 300,
+          "maxLength": 3000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "avatar": { "type": "string" },
+        "policies": {"type": "ref", "ref": "app.bsky.label.defs#labelerPolicies"},
+        "likeCount": { "type": "integer", "minimum": 0 },
+        "viewer": { "type": "ref", "ref": "#labelerViewerState" },
+        "indexedAt": { "type": "string", "format": "datetime" }
+      }
+    },
+    "labelerViewerState": {
+      "type": "object",
+      "properties": {
+        "like": { "type": "string", "format": "at-uri" }
+      }
+    },
+    "labelerPolicies": {
+      "type": "object",
+      "required": ["reportReasons", "labelValues"],
+      "properties": {
+        "description": {
```

Comment:
just so i understand: a labeler description will give a user-friendly high-level description of the labeler
and the policy description is the in the weeds more formal version right?

Reply:
Yep that's the logic